airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This module contains Google Dataprep hook.
+"""
+
+import requests
+from tenacity import retry, stop_after_attempt
+
+from airflow.providers.google.common.hooks.base_dataprep import DataprepBaseHook
+
+
+class GoogleDataprepHook(DataprepBaseHook):
+    """
+    Hook for connection with Dataprep API.
+
+    """
+
+    @retry(stop=stop_after_attempt(5))
+    def get_jobs_for_job_group(self, job_id: int):
+        """
+        Get information about the batch jobs within a Cloud Dataprep job.
+
+        :param job_id The ID of the job that will be fetched.
+        :type job_id: int
+        """
+        url: str = self.url + f"{job_id}/jobs"
```

Comment:
```suggestion
        url: str = f"{self.url}{job_id}/jobs"
```

Reply:
I would skip `/` from the `self.url` so I can make `f"{self.url}/{job_id}/jobs"`

Reply:
Good one, thx. :)